z_infra_provisioning/cloud_infra_center/ocp_upi/tools/modify-bastion.py

```diff
@@ -52,13 +52,14 @@ def get_nodes_ips(infra_id, node_role):
     nodes_dict = {}
     for node in nodes:
         name = node.split(" ")[0]
+        name = name.replace(infra_id, "")
         n = name.split("-")
-        if len(n) < 4:
+        if len(n) <= 1:
```

Comment:
> If len==1, it mean there is no "-" in name, why we break the loop here ? I think it will cause the next ip won't be configured.


It only break its loop itself, won't affect the next ports.

> Besides, IF there is other symbol, how we process it? such as "_" ...

If using "_", the ports ansible created are named like :
```
openshift_cluster-ktlh2-bootstrap-port
openshift_cluster-ktlh2-master-port-0
openshift_cluster-ktlh2-master-port-1
openshift_cluster-ktlh2-master-port-2
openshift_cluster-ktlh2-work-port-34dfr
```
However, 
1. I remove infra_id before splitting strings, so whatever the infra_id is, it won't affect the next splitting.
https://github.com/gyohuangxin/z_ansible_collections_samples/blob/0d412179f4214573db1f3b3dfb545f614c363d98/z_infra_provisioning/cloud_infra_center/ocp_upi/tools/modify-bastion.py#L55
2. we just use "-" to split above strings and get the result like "bootstrap", "master-0", "master-2"..
https://github.com/gyohuangxin/z_ansible_collections_samples/blob/0d412179f4214573db1f3b3dfb545f614c363d98/z_infra_provisioning/cloud_infra_center/ocp_upi/tools/modify-bastion.py#L56